plotter/boxplot_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/plot"
+	"github.com/gonum/plot/vg"
+)
+
+func ExampleBoxPlot() {
+	// Create the sample data.
+	rand.Seed(0) // The default random seed is 1.
```

Comment:
Since the example plots are not run by the testing harness without help from us, this can be factored out to checkPlot just prior to the call to the example function (and at the same time, may as well make it seed with 1).